onnx/defs/parser.cc

```diff
@@ -43,6 +43,35 @@ Status OnnxParser::Parse(char open, IdList& idlist, char close) {
   return Status::OK();
 }
 
+Status OnnxParser::Parse(IdList& idlist, AttrList& attrlist) {
+  idlist.Clear();
+  attrlist.Clear();
+  do {
+    std::string id;
+    bool is_assignment = PeekIsIdentifierAssignment(id);
+    if (id.empty())
+      return Status::OK(); // Treat as empty list of identifiers
+    if (is_assignment) {
+      PARSE(*attrlist.Add());
+    }
+    else {
+      ParseOptionalIdentifier(id);
+      *idlist.Add() = id;
+    }
+  } while (Matches(','));
+  return Status::OK();
+}
+
+Status OnnxParser::Parse(char open, IdList& idlist, AttrList& attrlist, char close) {
+  idlist.Clear();
+  attrlist.Clear();
```

Comment:
nit: The above two lines are duplicated in method above also. I wonder if we should just move this into an else-clause down below.